cmd/collectors/rest/plugins/snapmirror/snapmirror.go

```diff
@@ -182,9 +185,14 @@ func (my *SnapMirror) updateSMLabels(data *matrix.Matrix) {
 }
 
 func (my *SnapMirror) handleCGRelationships(data *matrix.Matrix, keys []string) {
-
 	for _, key := range keys {
 		cgInstance := data.GetInstance(key)
+		// find cgName from the destination_location
+		matches := cgNameRe.FindStringSubmatch(cgInstance.GetLabel("destination_location"))
+		if len(matches) == 2 {
+			cgInstance.SetLabel("cg_name", matches[1])
```

Comment:
I wonder if this would work as well and be clearer?
cgInstance.SetLabel("cg_name", filepath.Base(cgInstance.GetLabel("destination_location")))

Reply:
yes, much cleaner.